frontend/src/modules/scribble/utility/WebSocketMessageUtils.js

```diff
@@ -2,10 +2,6 @@ export function createChatMessage(message) {
   return createWebSocketMessage("chat", { message: message });
 }
 
-export function createDrawActionMessage(drawAction) {
```

Comment:
Maybe add something back in that sets the api to "draw" but forwards the payload along? e.g.: 
`export function createDrawMessage(payload) { return createWebSocketMessage("draw", payload);}`

Reply:
Hmm ok, i think that's helpful without abstracting away important details